### PR TITLE
Added index.html

### PR DESCRIPTION
**Pull request by amanmool03**
I have added the index.html with the header in it plesase review it

### CODE REVIEW

**Comment by amanmool03** [comment]
mnisha check the modification i did ??

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+<h1>Hello worldg</h1>
```
Comment (manesha07) [inline index.html]: hello world milena hai k ho yesto

File: index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+</body>
+</html>
```
Comment (amanmool03) [inline index.html:9]: what is this

File: index.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
```
Comment (manesha07) [inline index.html:4]: title birsyau hai title

File: index.html (hunk 2 above)
Comment (manesha07) [reply index.html:9]: i dunnooo what is it

File: index.html (hunk 3 above)
Comment (amanmool03) [reply index.html:4]: ok i will change it right away

Comment (manesha07) [reply index.html:4]: reallly?

**Comment by amanmool03** [comment]
i merged to the master branch

File: index.html (hunk 2 above)
Comment (manesha07) [reply index.html:9]: good job

**Pull request merged**